samples/drivers/video/capture/boards/stm32mp135f_dk.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_VIDEO_PIXEL_FORMAT="RGBP"
+CONFIG_VIDEO_FRAME_WIDTH=480
+CONFIG_VIDEO_FRAME_HEIGHT=272
+CONFIG_VIDEO_BUFFER_POOL_SZ_MAX=262000
+CONFIG_VIDEO_SOURCE_CROP_LEFT=80
+CONFIG_VIDEO_SOURCE_CROP_TOP=104
+CONFIG_VIDEO_SOURCE_CROP_WIDTH=480
+CONFIG_VIDEO_SOURCE_CROP_HEIGHT=272
```

Comment:
I think these were bothering CI but let's wait dependencies to be merged, maybe no need to debug CI until then...